README.md

```diff
@@ -132,6 +132,26 @@ To run ESLint automatically when a file is saved, run the following and leave it
 npm run eslint-watch
 ```
 
+### Building for Cesium integration
+
+Some functionality of gltf-pipeline is used by Cesium as a third party library. The necessary files can be generated using:
+
+```
+npm run build-cesium
+```
+
+This will output a portion of the gltf-pipeline code into the `dist/cesium` folder, reformatted into AMD style for use with RequireJS and Cesium in the browser.
+
+### Building for other integration
```

Comment:
Fine to remove this section since the `build-cesium-combine` task isn't used anymore.